docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/README.md

```diff
@@ -0,0 +1,122 @@
+# {{ cookiecutter.app_verbose_name }}
+
+## Preparing for Docker
+
+Before building the Docker container, run
+```bash
+mv env.example .env
+```
+
+## Developing
+
+Development requires a local installation of [Docker](https://docs.docker.com/install/)
+and [Docker Compose](https://docs.docker.com/compose/install/).
+
+Build application containers:
+
+```bash
+docker-compose up --build
+```
+
+To populate the starter content for the CMS, run:
+
+```bash
+docker-compose run --rm app python manage.py load_cms_content
```

Comment:
Hmm, I'm getting this:

```
django.core.serializers.base.DeserializationError: Problem installing fixture '/app/hannah/management/commands/../../fixtures/initial_cms_content.json':
```

Reply:
Pretty sure this is because I left the initial content pages completely blank. I've added an empty array there.